src/ext/VSExtension/wixext/VSHeatExtension.cs

```diff
@@ -11,6 +11,9 @@
 // </summary>
 //-------------------------------------------------------------------------------------------------
 
+using System.Net.Mime;
+using System.Text.RegularExpressions;
+
```

Comment:
Thanks Brian,

I will revise.
ReSharper jammed that Net.Mime in there.  I didn't notice the pollutuion.
I will correct this

regarding the Regex, I wanted to handle the case where there can be
semi-colons in the metadata filter.  For example, filtering on a HintPath
of C:\bin  could be used.
The Regex finds the first occurance.  Could certainly consider using .First
instead



On Tue, Mar 17, 2015 at 11:31 AM Brian Rogers <notifications@github.com>
wrote:

> In src/ext/VSExtension/wixext/VSHeatExtension.cs
> <https://github.com/wixtoolset/wix4/pull/110#discussion_r26601742>:
>
> > @@ -11,6 +11,9 @@
> >  // </summary>
> >  //-------------------------------------------------------------------------------------------------
> >
> > +using System.Net.Mime;
> > +using System.Text.RegularExpressions;
> > +
>
> Typically, as seen below, a using goes inside the namespace. Also, what is
> the purpose of System.Net.Mime here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wixtoolset/wix4/pull/110/files#r26601742>.
>